.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: php
-
+dist: bionic
 php:
-  - 7.2
-  - 7.3
   - 7.4
   - 8.0
+  - 8.1.0
```

Comment:
Do we need to pin to such a specific version?

Reply:
> Do we need to pin to such a specific version?

This is Travis issue. It does not work if the version is `8.1` https://app.travis-ci.com/github/wmde/Diff/jobs/563853202